modules/trust-cert-new-sensor.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * configuration/add-custom-certificates.adoc
+
+:_module-type: PROCEDURE
+[id="trust-cert-new-sensor_{context}"]
+= Configuring Sensor to trust custom certificates when deploying a new Sensor
+
+.Prerequisites
+
+* Make sure that you have downloaded the Sensor bundle.
+
+.Procedure
+
+* If you are using the `sensor.sh` script:
++
+. Unzip the Sensor bundle:
++
+[source,terminal]
+----
+$ unzip -d sensor sensor-<cluster-name>.zip
+----
+. Run the `sensor.sh` script:
++
+[source,terminal]
+----
+$ ./sensor/sensor.sh
+----
++
+The certificates are automatically applied when you run the sensor (`./sensor/sensor.sh`) script.
+You can also place additional custom certificates in the `sensor/additional-cas/` directory before you run the `sensor.sh` script.
+* If you are using the `sensor.sh` script:
```

Comment:
This second bullet says the same thing as the first one. Was the ". Without `sensor.sh` script:" supposed to be this second option?

Reply:
My bad, fixed it.